projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed-hlist-item/external-feed-hlist-item.component.html

```diff
@@ -35,4 +35,44 @@ <h3 class="name">
       </div>
     </nb-card-body>
   </nb-card>
```

Comment:
Why's all the above commented code present?

Reply:
Removed